api/src/wfl/server.clj

```diff
@@ -85,11 +86,34 @@
      (setDebugEnabled [enabled])
      (warn ([thrown]) ([msg & args])))))
 
+(defn ^:private start-workload-manager []
+  (letfn [(update-workloads! [])] ;; todo
+    (pprint "starting workload update loop")
+    (update-workloads!)
+    (future
+      (loop []
+        (update-workloads!)
+        (.sleep TimeUnit/SECONDS 20)
+        (recur)))))
+
+(defn ^:private start-server [args]
+  (let [port {:port (util/is-non-negative! (first args))}]
+    (pprint [wfl/the-name port])
```

Comment:
Same goes here. Not bad the way it is, just figure routing through `log` might help us redirect output more easily via the log4j config at some point